nupic/algorithms/anomaly_likelihood.py

```diff
@@ -84,18 +88,20 @@ def __init__(self,
     is that real anomalies might get ignored and not flagged.
 
     :param estimationSamples: the number of reasonable anomaly scores required
-    for the initial estimate of the Gaussian. The default of 300 records is
+    for the initial estimate of the Gaussian. The default of 100 records is
     reasonable - we just need sufficient samples to get a decent estimate for
     the Gaussian. It's unlikely you will need to tune this since the Gaussian is
-    re-estimated every 100 iterations.
+    re-estimated every 10 iterations by default.
 
     :param int estimationWindowSize: size of sliding window of historical
-      data points to maintain for periodic reestimation of the Gaussian. Note:
-      the default of 8640 is based on a month's worth of history at 5-minute
-      intervals.
-
-    Anomaly likelihood scores are reported at a flat 0.5 for claLearningPeriod +
-    estimationSamples iterations.
+    data points to maintain for periodic reestimation of the Gaussian. Note:
```

Comment:
I find it much easier to include indentation for continued param descriptions but I don't think there is an explicit convention.